src/modules/handlers/ManifestHandler.js

```diff
@@ -69,7 +69,7 @@ export async function StoreManifest(callback) {
 
     // Check if the manifest components are empty
     Object.keys(MANIFEST).forEach((key) => {
-      if (MANIFEST[key].length === 0) {
+      if (Object.keys(MANIFEST[key]).length === 0) {
```

Comment:
So this fails due to do it not being able to get the keys for an undefined object (from a comment on another commit). So to get this working I just added a check to make sure it's not undefined. `(MANIFEST[key] && Object.keys(MANIFEST[key]))`

Reply:
Alright figured it out, just needs to be replaced with `if (!MANIFEST[key] || Object.keys(MANIFEST[key]).length === 0) {`